application/sitebuilder/build_service.py

```diff
@@ -35,6 +36,11 @@ def request_build():
 
 
 def build_site(app):
+    def print_stacktrace():
+        traceback.print_stack()
```

Comment:
We probably only want this if the exit code isn't zero?  But don't know how/if you can grab the code in the `atexit()`

Reply:
Yeah, we can always remove it later?